tests/test_unittest/test_gtsimport.py

```diff
@@ -0,0 +1,305 @@
+import sys
+
+import pytest
+
+from gt4py import gtsimport
+
+
+@pytest.fixture(scope="function")
+def reset_importsys():
+    print("storing data")
```

Comment:
Here the print statements intended here?

Reply:
Indeed, pytest has this nice way of capturing output and showing it to you only if something goes wrong.
However, as I am looking at this I see it is not very helpful unless you know what's going on already. I am thinking about making the message clearer and possibly replacing it with a logging call so it can be switched on/off easily.